src/components/SidePanel/styles.module.scss

```diff
@@ -0,0 +1,35 @@
+@import '../../scss/variables';
+
+.ToggleButton {
+  height: auto;
+  overflow: hidden;
+  padding: $default-padding;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  width: 100%;
+
+  &:not(:global(.focus-visible)) {
+    box-shadow: none;
+  }
+}
+
+.content {
+  overflow-x: auto;
+}
+
+.isCollapsed {
+  background-color: $light-gray;
+  grid-gap: 0;
+  // TODO: find a way to avoid the `!important` here. There is a conflict with
```

Comment:
You were only seeing this in the storybook, right? Building on the patch I suggested up above, this seems like an acceptable solution. Does that work?

```diff
diff --git a/src/components/FullscreenGrid/styles.module.scss b/src/components/FullscreenGrid/styles.module.scss
index 7a1e0e8..a93beda 100644
--- a/src/components/FullscreenGrid/styles.module.scss
+++ b/src/components/FullscreenGrid/styles.module.scss
@@ -30,15 +30,6 @@ $expand-button-width: 40px;
   grid-area: FGHeader;
 }
 
-.mainSidePanel,
-.altSidePanel {
-  display: grid;
-  flex-direction: column;
-  grid-gap: $default-padding;
-  grid-template-rows: 1fr auto;
-  height: auto;
-}
-
 .mainSidePanel {
   grid-area: FGMainSidePanel;
   margin-left: $default-padding;
@@ -60,5 +51,8 @@ $expand-button-width: 40px;
   border: 1px solid $light-gray;
   overflow-x: hidden;
   overflow-y: auto;
+}
+
+.content {
   padding: $default-padding;
 }
diff --git a/src/components/SidePanel/index.tsx b/src/components/SidePanel/index.tsx
index e6f909b..dd084a3 100644
--- a/src/components/SidePanel/index.tsx
+++ b/src/components/SidePanel/index.tsx
@@ -26,7 +26,7 @@ export const SidePanelBase = ({
   return (
     <aside
       aria-expanded={isExpanded ? 'true' : 'false'}
-      className={makeClassName(className, {
+      className={makeClassName(styles.panel, className, {
         [styles.isCollapsed]: !isExpanded,
       })}
     >
diff --git a/src/components/SidePanel/styles.module.scss b/src/components/SidePanel/styles.module.scss
index 528a719..3a300ad 100644
--- a/src/components/SidePanel/styles.module.scss
+++ b/src/components/SidePanel/styles.module.scss
@@ -1,5 +1,13 @@
 @import '../../scss/variables';
 
+.panel {
+  display: grid;
+  grid-gap: $default-padding;
+  grid-template-rows: 1fr auto;
+  height: auto;
+  padding: $default-padding;
+}
+
 .ToggleButton {
   height: auto;
   overflow: hidden;
@@ -20,9 +28,7 @@
 .isCollapsed {
   background-color: $light-gray;
   grid-gap: 0;
-  // TODO: find a way to avoid the `!important` here. There is a conflict with
-  // https://github.com/mozilla/addons-code-manager/blob/38f50dfb68ee031b518fe2fb38afa6a705aec461/src/components/FullscreenGrid/styles.module.scss#L64
-  padding: 0 !important;
+  padding: 0;
 
   .ToggleButton {
     height: 100%;
```

Reply:
> You were only seeing this in the storybook, right? Building on the patch I suggested up above, this seems like an acceptable solution. Does that work?

Yep, it does! Thanks 🥇